exercises/practice/triangle/.meta/tests.toml

```diff
@@ -1,60 +1,73 @@
+# - Preserve any other key/value pair
+#
+# As user-added comments (using the # character) will be removed when this file
+# is regenerated, comments can be added via a `comment` key.
 
 [8b2c43ac-7257-43f9-b552-7631a91988af]
-description = "all sides are equal"
+description = "equilateral triangle -> all sides are equal"
 
 [33eb6f87-0498-4ccf-9573-7f8c3ce92b7b]
-description = "any side is unequal"
+description = "equilateral triangle -> any side is unequal"
 
 [c6585b7d-a8c0-4ad8-8a34-e21d36f7ad87]
-description = "no sides are equal"
+description = "equilateral triangle -> no sides are equal"
 
 [16e8ceb0-eadb-46d1-b892-c50327479251]
-description = "all zero sides is not a triangle"
+description = "equilateral triangle -> all zero sides is not a triangle"
 
 [3022f537-b8e5-4cc1-8f12-fd775827a00c]
-description = "sides may be floats"
+description = "equilateral triangle -> sides may be floats"
 
 [cbc612dc-d75a-4c1c-87fc-e2d5edd70b71]
-description = "last two sides are equal"
+description = "isosceles triangle -> last two sides are equal"
 
 [e388ce93-f25e-4daf-b977-4b7ede992217]
-description = "first two sides are equal"
+description = "isosceles triangle -> first two sides are equal"
 
 [d2080b79-4523-4c3f-9d42-2da6e81ab30f]
-description = "first and last sides are equal"
+description = "isosceles triangle -> first and last sides are equal"
 
 [8d71e185-2bd7-4841-b7e1-71689a5491d8]
-description = "equilateral triangles are also isosceles"
+description = "isosceles triangle -> equilateral triangles are also isosceles"
 
 [840ed5f8-366f-43c5-ac69-8f05e6f10bbb]
-description = "no sides are equal"
+description = "isosceles triangle -> no sides are equal"
 
 [2eba0cfb-6c65-4c40-8146-30b608905eae]
-description = "first triangle inequality violation"
+description = "isosceles triangle -> first triangle inequality violation"
 
 [278469cb-ac6b-41f0-81d4-66d9b828f8ac]
-description = "second triangle inequality violation"
+description = "isosceles triangle -> second triangle inequality violation"
 
 [90efb0c7-72bb-4514-b320-3a3892e278ff]
-description = "third triangle inequality violation"
+description = "isosceles triangle -> third triangle inequality violation"
 
 [adb4ee20-532f-43dc-8d31-e9271b7ef2bc]
-description = "sides may be floats"
+description = "isosceles triangle -> sides may be floats"
 
 [e8b5f09c-ec2e-47c1-abec-f35095733afb]
-description = "no sides are equal"
+description = "scalene triangle -> no sides are equal"
 
 [2510001f-b44d-4d18-9872-2303e7977dc1]
-description = "all sides are equal"
+description = "scalene triangle -> all sides are equal"
 
 [c6e15a92-90d9-4fb3-90a2-eef64f8d3e1e]
-description = "two sides are equal"
+description = "scalene triangle -> first and second sides are equal"
+
+[3da23a91-a166-419a-9abf-baf4868fd985]
+description = "scalene triangle -> first and third sides are equal"
+
+[b6a75d98-1fef-4c42-8e9a-9db854ba0a4d]
+description = "scalene triangle -> second and third sides are equal"
```

Comment:
This says there are additional tests being added. Did you add those tests to the unit test file?

Reply:
I might not have done so, I ran the confliglet commands and checked the files to ensure nothings was changed in the config file. Would that be the implementation step?

Reply:
I'm not sure what set of steps you're following to comment on what it would be labeled. 

These UUIDs map to test cases in [the canonical data](https://github.com/exercism/problem-specifications/blob/main/exercises/triangle/canonical-data.json) and need to line up with [the C++ test cases](https://github.com/exercism/cpp/blob/main/exercises/practice/triangle/triangle_test.cpp). The ask isn't to run a command. The ask is to update the test files -- both the toml metadata and the unit tests. They need to be done at the same time and be kept in sync.

Reply:
Hi, I apologize as my answer wasn't clear. I did try to update the tests. I will rectify this and update my PR. Thank you.

Reply:
Are you still working on this @DebbieSan? Do you need help?

Reply:
Hi @vaeng, I did get caught up with school but I am working on this. I will pick it back up today. I have a question though about the tests. 

```
// canonical-data.json tests this with a scalene triangle?
TEST_CASE("first_and_third_sides_are_equal")
{
    REQUIRE(triangle::flavor::isosceles == triangle::kind(3, 4, 3));
}

// canonical-data.json tests this with a scalene triangle?
TEST_CASE("second_and_third_sides_are_equal")
{
    REQUIRE(triangle::flavor::isosceles == triangle::kind(4, 3, 3));
}
```

The triangles in the test case have property:scalene but they are isosceles. Is this correct? Are the tests just suppose to give a true/false state? 

Thank you for your assistance.

Reply:
Yes, the REQUIRE Test macro expects a true or false inside the brackets, so most tests use a `==` comparison between an expected value and the one the student-implemented function returns.

Reply:
@DebbieSan 
Are you good to move on, or do you have more questions?

Reply:
Hello, I updated the PR.